src/utils/ExtensionActivationManager.ts

```diff
@@ -0,0 +1,86 @@
+/*!--------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+const builtInExtensionIdRegex = /^vscode\./i;
+
+const activationEventRegex = /^x-onAzNodeType(?<trigger>Fetch|Resolve):(?<type>[\w.\/]+)$/;

```

Comment:
Should this maybe be `x-ms-onAzNodeTypeFetch`? With `-ms`.

Reply:
I don't have a preference, I think the `x-` is enough, but also no harm in adding the `-ms`.